docs/strategies/timeout.md

```diff
@@ -119,3 +124,51 @@ sequenceDiagram
     T->>P: Throws <br/>TimeoutRejectedException
     P->>C: Propagates exception
 ```
+
+## Anti-patterns
+
+### Ignoring Cancellation Token
+
+❌ DON'T
+
+Ignore the cancellation token provided by the resilience pipeline:
+
+<!-- snippet: timeout-ignore-cancellation-token -->
+```cs
+var outerToken = CancellationToken.None;
+
+var pipeline = new ResiliencePipelineBuilder()
+    .AddTimeout(TimeSpan.FromSeconds(1))
+    .Build();
+
+await pipeline.ExecuteAsync(
+    async innerToken => await Task.Delay(3000, outerToken), // The delay call should use innerToken
```

Comment:
In every other places we use `TimeSpan.FromXYZ` in the examples. I would suggest to use here as well.

```suggestion
    async innerToken => await Task.Delay(TimeSpan.FromSeconds(3), outerToken), // The delay call should use innerToken
```